src/lib/krb5/krb/ser_actx.c

```diff
@@ -319,25 +234,22 @@ krb5_auth_context_externalize(krb5_context kcontext, krb5_pointer arg, krb5_octe
 
 /* Internalize a keyblock and convert it to a key. */
 static krb5_error_code
-intern_key(krb5_context ctx, krb5_key *key, krb5_octet **bp, size_t *sp)
+intern_key(krb5_key *key, krb5_octet **bp, size_t *sp)
 {
     krb5_keyblock *keyblock;
     krb5_error_code ret;
 
-    ret = krb5_internalize_opaque(ctx, KV5M_KEYBLOCK,
-                                  (krb5_pointer *) &keyblock, bp, sp);
+    ret = k5_internalize_keyblock(&keyblock, bp, sp);
     if (ret != 0)
         return ret;
-    ret = krb5_k_create_key(ctx, keyblock, key);
-    krb5_free_keyblock(ctx, keyblock);
+    ret = krb5_k_create_key(NULL, keyblock, key);
```

Comment:
It doesn't cause problems here, but is this something we're okay with doing in general in internal code?

Reply:
Yes.  I have plans to document that contexts are optional in many API functions.  That's not a commitment to never use the context, just to behave reasonably when there isn't one.  In the meantime I think it's fine to pass null internally (and we already do in many places).